.github/workflows/migrations.yml

```diff
@@ -0,0 +1,21 @@
+name: commcare-hq migrations check
+on:
+  pull_request:
+    branches:
+    - master
+    - hotfix-deploy
```

Comment:
do we ever use this branch? I would have thought it would always be a dynamic name like `hotfix-2021-12-01`

Reply:
(Interesting, I see we have it documented like this on the wiki: https://confluence.dimagi.com/display/GTD/Deploy+Process)

Reply:
I think there is a new workflow in place to start using it more. I'll admit I just copied this from `.github/workflows/tests.yml`, but I did notice it and assumed it's valid.